packages/headless/src/controllers/search-box/headless-search-box.ts

```diff
@@ -77,6 +78,16 @@ export function buildSearchBox(
     })
   );
 
+  const getValue = () => engine.state.querySet[options.id];
+
+  const submit = (analytics: SearchAction) => {
```

Comment:
Small nitpick, but I find it a bit weird to have a `submit` internal function, and then another external function called `submit`, it can be a bit confusing ?

But I'm not sure how to call it either....

Suggestions: 
`performSearch` 
`performQuery`
`doQuery`